competitions/LLIIA2020/ansible/inv/local/group_vars/all.tmpl

```diff
@@ -64,3 +64,6 @@ simplesaml_install_directory: ${SIMPLESAML_INSTALL_DIRECTORY}
 # See https://developer.okta.com/code/php/simplesamlphp/ for more information
 simplesaml_okta_metadata_name: ${SIMPLESAML_OKTA_METADATA_NAME}
 simplesaml_okta_metadata_url: ${SIMPLESAML_OKTA_METADATA_URL}
+
+# The version of SimpleBook to use in this competiton
+simplebook_git_version: main
```

Comment:
We're *almost* ready to lock in a git version of SimpleBook; I'd like to get through this first round of changes and release a v1 on the SimpleBook side first.